### PR TITLE
fix(65): Setting variable does not work when value has dash: Updates …

### PR DESCRIPTION
**Pull request by AndrewAllison**
My first PR so please be gentle 😁 

I've added a new regex. It might be far from the greatest approach but seemed like a quick fix for something that's causing me and others issues. 

Tested against 
```shell
node cli.js -v test="$(echo bax; echo foo=bar)" -p test
node cli.js -v test="123-4543-213" -p test
```

Also added examples https://regexr.com/6i33o

### CODE REVIEW

File: cli.js [hunk 1]
```diff
@@ -47,7 +47,7 @@ if (argv.c) {
 }
 
 function validateCmdVariable (param) {
-  if (!param.match(/^\w+=\w+$/)) {
+  if (!param.match(/^\w+=[a-zA-Z0-9"=^!?%@_&-/:;.]+$/)) {
```
Comment (entropitor) [inline cli.js:50]:
```suggestion
  if (!param.match(/^\w+=[a-zA-Z0-9"=^!?%@_&\-/:;.]+$/)) {
```
The `=` might cause problems when not within `"` but 🤷 

You missed an escape before the `-` literal so I suggest to add that

**Review by entropitor — APPROVED** [review]

**Comment by entropitor** [comment]
Published as v5.1.0

**Comment by klarstrup** [comment]
nice

**Comment by domneuk** [comment]
First of all: thank you for your work! 

I stumbled upon a similar problem: Adding `DEBUG=release-it:*` fails, because it has a star in it.

Can you please add this the regex?

**Pull request merged**